bedrock/contentful/api.py

```diff
@@ -477,8 +481,27 @@ def _get_preview_image_from_fields(self, fields):
             if preview_image_url:
                 return f"https:{preview_image_url}"
 
+    def _get_image_from_default_locale_seo_object(
+        self,
+        contentful_id,
+        default_locale=DEFAULT_LOCALE,
+    ):
+        try:
+            entry = ContentfulEntry.objects.get(
+                contentful_id=contentful_id,
+                locale=default_locale,
+            )
+        except ContentfulEntry.DoesNotExist:
+            return ""
+        return entry.data.get("info", {}).get("seo", {}).get("image", None)
```

Comment:
Just noticing the difference between an empty db hit returning an empty string, vs the db hit possibly returning a `None`. (Also, `None` is the default and is not needed there)

Reply:
Yeah, I wanted it to be explicit about the `None`, but I get your point. It might be better overall to reutrn `""` so that it's cosistent regardless of DB state, yeah?

Reply:
It's definitely a preference. I was on a team that very much disliked being explicit if it was the default, so I guess I got used to doing the same.